src/main/java/io/dataspaceconnector/services/usagecontrol/PolicyExecutionService.java

```diff
@@ -50,19 +57,55 @@ public void logDataAccess(final String target) throws PolicyExecutionException {
         }*/
     }
 
+
+  /**
+   * Send a message to the URL. Allow the access only if that operation was successful.
+   *
+   * @param target The target object.
+   * @throws PolicyExecutionException if the access could not be successfully logged.
+   */
+  public void remoteNotifyDataAccess(final String target) throws PolicyExecutionException {
```

Comment:
Do you have any reason for keeping this comment?
Is the code tested?

Reply:
It is still in progress and hasn't been finished yet.